troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Path Too Long Exception When Building Project in Microsoft Visual Studio
+description: AppBuilder projects with fully qualified file names longer that 260 symbols fail to build in the extension for Visual Studio.
```

Comment:
Leave it only: "AppBuilder projects with fully qualified file names longer that 260 symbols fail to build."